cmd/broker.go

```diff
@@ -64,12 +73,20 @@ func init() {
 	brokerCmd.PersistentFlags().StringVarP(&brokerName, "name", "n", "", "Name of Automation Broker instance")
 	rootCmd.AddCommand(brokerCmd)
 
+	brokerCatalogCmd.Flags().StringVarP(&catalogOutputFormat, "output", "o", "", "Display broker catalog output in JSON or YAML format")
 	brokerCmd.AddCommand(brokerCatalogCmd)
 
 	brokerCmd.AddCommand(brokerBootstrapCmd)
 	rootCmd.AddCommand(createHiddenCmd(brokerBootstrapCmd, "running 'apb broker bootstrap'"))
 }
 
+func pruneOutputFormat(format string) {
+	if format != "yaml" && format != "json" && format != "" {
+		log.Warnf("Did not recognize --output argument [%v], printing as table", format)
```

Comment:
Could we give the user a list of acceptable output types here? Let them know that they must type either "json" or "yaml".